package/babel.config.js

```diff
@@ -19,22 +20,19 @@ const presets = [
 
 const plugins = [
   [
-    require.resolve('babel-plugin-module-resolver'),
+    require.resolve("babel-plugin-module-resolver"),
     {
-      root: ['./src/'],
+      root: ["./src/"],
       alias: {
-        '~': './src/data',
-        '@files': './src/files.js',
-        '@filesObjects': './src/filesObjects.js',
-        '@utils': './src/utils.js',
-        '@generator': './generator'
+        "~": "./src/data",
+        "@files": "./src/files.js",
+        "@utils": "./src/utils.js",
+        "@generator": "./generator"
```

Comment:
can you remove this line - we don't need it anymore i think

Reply:
this alias is used for our test